src/upload-tests.proj

```diff
@@ -7,20 +7,27 @@
          - Add support for storing test-project-specific timeouts, use this instead of default if available
   -->
 
-  <UsingTask TaskName="ZipFileCreateFromDirectory"       AssemblyFile="$(BuildToolsTaskDir)Microsoft.DotNet.Build.Tasks.dll"/>
+  <UsingTask TaskName="SimulateHelixExecution"     AssemblyFile="$(BuildToolsTaskDir)Microsoft.DotNet.Build.CloudTestTasks.dll"/>
+  <UsingTask TaskName="ZipFileCreateFromDirectory" AssemblyFile="$(BuildToolsTaskDir)Microsoft.DotNet.Build.Tasks.dll"/>
 
   <PropertyGroup>
     <!-- Workaround for dealing with building on Linux/OSX.  
          It doesn't seem possible to pass a connection string on the command line, as it contains ';' characters. 
          Checking for the existence of a connection string will happen inside CloudTest.Helix.targets -->
     <CloudDropConnectionString    Condition="'$(CloudDropConnectionString)'==''"   >DefaultEndpointsProtocol=https;AccountName=$(CloudDropAccountName);AccountKey=$(CloudDropAccessToken);EndpointSuffix=core.windows.net</CloudDropConnectionString>
     <CloudResultsConnectionString Condition="'$(CloudResultsConnectionString)'==''">DefaultEndpointsProtocol=https;AccountName=$(CloudResultsAccountName);AccountKey=$(CloudResultsAccessToken);EndpointSuffix=core.windows.net</CloudResultsConnectionString>
-    
+
+    <!-- Set this to true to simulate Helix execution.  Will fail if Python 2.7.X is unavailable 
+         We'll also opportunistically set SkipSendToHelix true here, since if the user is doing 
+         local execution they likely don't want to run in the cloud. -->
+    <LocalHelixExecution>false</LocalHelixExecution>
```

Comment:
it's true that if you pass it like `/p` it would still work, but perhaps somebody might be testing locally and don't want to have to pass in this value everytime so they just set an environment variable, and in that case it wouldn't work. I don't feel this would be a problem unless somebody for whatever reason decides to set this to true unintentionally in some props file.

That said, I'm also not totally against of how you have it already, so I'm fine with keeping it as is as well.